codeland.html

```diff
@@ -1311,7 +1331,7 @@ <h4 class="codeland-schedule__event-title">
     presentSidecar(
       event.target.dataset.url,
       'Social Corner',
-      'Find all these articles in <a href="/t/codeland" target="_blank">#codeland</a>',
+      'Powered by <a href="/connect" target="_blank">Connect</a>',
```

Comment:
Nice :)

Reply:
I remember we had this already and I may have accidentally overrode it at some point 😅